From c5f942b863c660280535dac32586e64d45308550 Mon Sep 17 00:00:00 2001
Message-ID: <c5f942b863c660280535dac32586e64d45308550.1736748624.git.sam@gentoo.org>
In-Reply-To: <9f57274faacf1dbb6c2b56d29de150b5e79ecb3d.1736748624.git.sam@gentoo.org>
References: <9f57274faacf1dbb6c2b56d29de150b5e79ecb3d.1736748624.git.sam@gentoo.org>
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Tue, 27 Oct 2020 06:39:59 -0700
Subject: [PATCH 8/8] h8300: Handle .gnu_object_only section

	PR ld/12291
	PR ld/12430
	PR ld/13298
	* config/tc-h8300.c (h8300_elf_section): Handle .gnu_object_only
	section.
---
 gas/config/tc-h8300.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/gas/config/tc-h8300.c b/gas/config/tc-h8300.c
index f8e54064166..105461da92b 100644
--- a/gas/config/tc-h8300.c
+++ b/gas/config/tc-h8300.c
@@ -146,8 +146,10 @@ pint (int arg ATTRIBUTE_UNUSED)
 static void
 h8300_elf_section (int push)
 {
-  static const char * known_data_sections [] = { ".rodata", ".tdata", ".tbss" };
-  static const char * known_data_prefixes [] = { ".debug", ".zdebug", ".gnu.warning" };
+  static const char * known_data_sections []
+    = { ".rodata", ".tdata", ".tbss", ".gnu_object_only" };
+  static const char * known_data_prefixes []
+    = { ".debug", ".zdebug", ".gnu.warning" };
   char * saved_ilp = input_line_pointer;
   const char * name;
 
-- 
2.48.0

